modules/apps/frontend-css/frontend-css-custom-properties-sample-web/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,14 @@
+aspect-ratios=Aspect Ratios
+borders=Borders
+box-shadow=Box Shadow
+css-custom-properties-sample=CSS Custom Properties Sample
+displays=Displays
+font-families=Font Families
+font-weights=Font Weights
+grays=Grays
+headings=Headings
+others=Others
+spacers=Spacers
+the-quick-brown-fox-jumps-over-the-lazy-dog=The quick brown fox jumps over the lazy dog
```

Comment:
Do you think this one can be translated? 🤔

Reply:
I thought some languages like arabic, japanese, russian, etc... could make a good use of this possibility

Reply:
Yeah that's what came to mind... but I'm not sure how realistic this is... it implies that translators have typography knowledge and know which phrase to use for their specific language. What I guess will happen most likely is that translators will simply translate this verbatim, which will probably make no sense at all 😂